.github/workflows/destroy.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   destroy:
-    if: ${{ github.actor != 'dependabot[bot]' }}
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'dependencies')) }}
```

Comment:
There is an argument of creating review apps for dependency bumps to ensure we have an extra step to test, but depends on what the wider team thinks

Reply:
I'm not sure it's worth it for dependencies. Every now and then there might be a special case (Rails) but we can always just create an ad hoc feature branch in those circumstances.